IFC4x3/Properties/l/LoadBearingCapacity_2lYlgvVFXB3O5r14yZzVUv/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LoadBearingCapacity_2lYlgvVFXB3O5r14yZzVUv" Name="LoadBearingCapacity" UniqueId="af8afab9-7cf8-4b0d-8175-044f23f5f7b9" PrimaryDataType="IfcPlanarForceMeasure" />
```

Comment:
Reuse?

Reply:
Done, reused of concept from footing. 6f854513e76831034aa88e2e9d239a8b55e47886